metagamer/environments/tictactoe.py

```diff
@@ -17,6 +17,28 @@
 logger = logging.getLogger(__name__)
 
 
+def policy_page_lines(board: np.array, player: int = 1) -> tuple:
+    """
+    return a possible position that hasn't yet been played.
+    This policy tries simply to fill the board from the top left.
+    Args:
+        board: a 3*3 numpy array whose elements are 0, 1, -1
+        player: either 1 or -1, representing
+    representing empty, player1 and player 2,
```

Comment:
issue: this line appears to be part of the definition for board. 

suggestion: Given that board is defined in module docstring, you can just skip over the doc or say as defined in module

Reply:
agreed